code/datums/gamemodes/flock.dm

```diff
@@ -4,6 +4,7 @@
 
 	shuttle_available = 2
 	shuttle_available_threshold = 12000 // 20 min, default value, probably change this
+	shuttle_unavailable_msg = "CentCom reports extreme anomalous radio signals originating from your station. The shuttle has been returned to base as a precaution."
```

Comment:
Since shuttle_available is always `2` for this game mode, this message most likely never appears.